client/src/components/_common/ReadMore/ReadMore.module.scss

```diff
@@ -1,6 +1,12 @@
 @import '../../../styles/tools/mixins.scss';
 
-.text-container {
+.clamped {
   overflow: hidden;
-  text-overflow: ellipsis;
-}
\ No newline at end of file
+  display: -webkit-box;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
```

Comment:
__Notice__

_Short Version._ Though [I warn against this, my own suggested CSS](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1611856604054700?thread_ts=1611853875.046800&cid=CQUA4D5KJ), my research and testing suggests it's okay.

<details><summary><em>Long Version</em></summary>

I had warned [not to use my CSS](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1611856604054700?thread_ts=1611853875.046800&cid=CQUA4D5KJ) but you probably didn't notice the message. I've done some research. Let's use it, anyway. The `-webkit-box-orient` is supposedly outdated and not supported or soon to be not supported... but it seems a new `line-clamp` is coming that negates the need for it. Also, all major browsers (even Firefox, regardless what caniuse.com says) supports this code.

</details>